### DISCUSSION TITLE
".asPlayer()"

### DISCUSSION THREAD

**Discussion by Calc-oholic** [Minestom/Minestom#1945] [Help] [answered]
Always says it's deprecated. I'm mainly using it in this context: "sender.asPlayer()". What should I use instead?

**Comment by tahmid-23** [comment, marked as answer]
```java
if (sender instanceof Player player) {
    // use the player
}
```

**Reply by TheMode** [reply]
Available since jdk16 https://openjdk.org/jeps/394
